src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
according to naming convention enum should be singular
```suggestion
public enum Color {
```